src/components/header.html

```diff
@@ -1,164 +1,53 @@
-<tr>
-  <td class="p-16">
-    <div class="flex flex-row items-center space-x-8">
+<table class="w-full mt-32 sm:mt-0">
+  <tr class="inline">
+    <td class="items-center align-middle py-16 pl-16">
       <a href="https://sealcred.xyz" class="no-underline">
-        <svg
+        <img
```

Comment:
Is using the img the reason why it doesn't show up on the email?

Reply:
Instead of the svg

Reply:
SVGs doesn't work in most email clients

Reply:
https://stackoverflow.com/questions/37753911/how-can-i-embed-svg-into-html-in-an-email-so-that-its-visible-in-most-all-emai